app/renderer/components/Inspector/SessionInfo.js

```diff
@@ -0,0 +1,128 @@
+import React, { Component } from 'react';
+import { Table, Card } from 'antd';
+import SessionCodeBox from './SessionCodeBox';
+import { withTranslation } from '../../util';
+
+const sessionObj = {
+  session_id: 'Session ID', session_url: 'Session URL',
+  server_details: 'Server Details', session_length: 'Session Length',
+  session_caps: 'Session Capabilities', server_response: 'Server Details from Server',
+  active_appId: 'Currently Active App ID'
+};
+
+const distanceY = 105;
```

Comment:
`DISTANCE_Y` isn't super explanatory. when i come to read this file and see this constant at the top, it would be nice to know a bit more about what it's for in the name.